ui/app/i18n/clinical/locale_pt.json

```diff
@@ -365,6 +365,9 @@
   "FOLLOWUP_CLINICAL_CONSULTATION": "Consulta Clínica",
   "FOLLOWUP_CLINICAL_CONSULTATION_A": " Clínica - Consulta de Seguimento",
   "ORDER_TYPE": "Exames",
+  "Panels": "Categorias",
+  "Lab Tests": "Exames",
+  "Terms which don't fit other categories": "Termos que não se enquadram em outras categorias",
```

Comment:
Use a standard format for key 
- don't use space between words in key

Reply:
Hi @sushantpatil1214,  Those words are coming from the database! Any tip on how i can go through this?